connectionchecker/connection_checker.go

```diff
@@ -206,3 +209,20 @@ func (cc *ConnectionChecker) checkProxySQLConnection(ctx context.Context, dsn st
 
 	return &res
 }
+
+func (cc *ConnectionChecker) checkExternalConnection(ctx context.Context, uri string) *agentpb.CheckConnectionResponse {
+	var res agentpb.CheckConnectionResponse
+
+	resp, err := http.Get(uri)
```

Comment:
maybe it's better to use `NewRequestWithContext` and have some timeout here. what do you think?